client/src/unifyfs.c

```diff
@@ -2431,16 +2455,6 @@ int unifyfs_mount(const char prefix[], int rank, size_t size,
     bool b;
     char* cfgval;
 
-    if (-1 != unifyfs_mounted) {
```

Comment:
do we need to drop this check?

Reply:
Oops, I must have removed this while debugging something.  I'll add it back in